.github/workflows/ci.yaml

```diff
@@ -11,15 +11,16 @@ jobs:
     strategy:
       matrix:
         node-version: [12.x]
+        rust: ['1.58.1']
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
       - name: Install Rust
         run: |
-          rustup update 1.52.1 --no-self-update
-          rustup default 1.52.1
+          rustup update ${{ matrix.rust }} --no-self-update
+          rustup defaul ${{ matrix.rust }}
```

Comment:
```suggestion
          rustup default ${{ matrix.rust }}
```

Reply:
Good catch, and curse the Mac double-click.